### PR TITLE
[ios-playbook] TECH Renew development certificates and provisioning profiles section

### PR DESCRIPTION
**Pull request by ldi13**
## Description
Add new section to follow the procedure to renew development certificates and provisioning profiles.

### CODE REVIEW

File: Utilities/CertsPP.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+# Renew Development Certificates and Provisioning Profiles for Development 
+
+Suppose that you have development certificate(s) that expire(s). As soon as it comes to you:
+
+* Your Provisioning Profiles related to it (them) become unusable
+* You have no possibility to deploy your daily basis code on test devices
+
+Here the procedure to follow to renew all that "stuff".
```
Comment (ghost) [inline Utilities/CertsPP.md]: Here `is` the procedure

File: Utilities/CertsPP.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    force_for_new_devices: true,
+    readonly: false,
+    force: true
+  )
+end
+
+```
+## Parameters
+* **\##BRANCH\_TO\_TARGET\##:** The targeted git branch on iOS-Certificates repository (`enterprise` vs `appstore`)
+* **\##TYPE\##**: Three types of environment: `development`, `enterprise`, `appstore`
```
Comment (ghost) [inline Utilities/CertsPP.md:39]: dev`e`lopment ?

File: Utilities/CertsPP.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+* **\##BUNDLE\_IDENTIFIER\##:** The bundle identifier for which you wish to generate another Provisioning Profile (`com.cp-driver.staging` for example)
+* **\##TEAM_ID\##**: The Team ID referenced on iOS Dev Center, and under which the Provisioning Profile has been defined
+
+## NB
+_**Take good care of referencing appropriate bundle identifiers considering the environment on which you want to renew the Certificates / Provisioning Profiles.**_
+
+_**Before wiping them from Git repository, identify which Provisioning Profiles and Certificates are related.**_
+
+_**For enterprise / appstore environments, please consult an helder team mate of yours**_ 👴🏼
```
Comment (ghost) [inline Utilities/CertsPP.md]: `elder teammate` :)

**Review by ghost — APPROVED** [review]
Some typos but otherwise great work 👏. Thank you for taking the time to solve an important issue but also document on how to do it. 🙇

File: Utilities/CertsPP.md (hunk 2 above)
Comment (ldi13) [reply Utilities/CertsPP.md:39]: `e` already there no?

File: Utilities/CertsPP.md (hunk 3 above)
Comment (ldi13) [reply Utilities/CertsPP.md]: 👍

File: Utilities/CertsPP.md (hunk 2 above)
Comment (ghost) [reply Utilities/CertsPP.md:39]: I swear it wasn't there 🤔

**Pull request merged**